### DISCUSSION TITLE
hru joabo

### DISCUSSION THREAD

**Discussion by segman1** [thejoabo/virtualfisher-bot#101] [General]
Hello, I hope you're doing well. I was just curious to know how things are going with you. Also, I remember you mentioning a new project you were working on a while back, and I'm curious about its progress. How is it coming along?

Anyway, I hope you're doing okay and staying safe. Take care

Best regards,
segman1

**Comment by thejoabo** [comment]
@segman1 Hello, thx for asking. I'm ok, really busy tho. Although I love these side projects, I have to take care of IRL stuff first, currently I'm working and finishing my degree, so I need to prioritize the things that pays my bills. This project is not abandoned tho, I'll be updating and adding new features when I have free time to spare.
The same thing applies to that other project, it's paused right now until I have the time to do so.

**Reply by segman1** [reply]
if you don't mind me asking what's the other project abt? also good 2 hear that you're doing well.

**Reply by thejoabo** [reply]
Sure, I don't know if yo are familiar with the game "Black Desert", but it has a very rich and complex central market, in which the player can buy and sell goods for the price they want (restricted by the min-max imposed by the game). The idea was to create yet another automation tool for the market to carry out automatic operations based on market analysis, to help players analyze trends and timings to prepare for a surge in demand, for example.
They already have a market API that can be accessed over the browser, but its too clunky to use and very restricted, my goal was to "reverse engineer" their API logic, so that I can understand and bypass those restrictions (of automations and what not). Also, I wanted to work with databases (to have a offline workplace), with a proper GUI (using pyqt or tkinter) and a statistical approach for market analysis.
